server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -120,6 +120,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion DESIRED_NODE_VERSION_OPTIONAL_STRING = def(8_580_00_0);
     public static final TransportVersion ML_INFERENCE_REQUEST_INPUT_TYPE_UNSPECIFIED_ADDED = def(8_581_00_0);
     public static final TransportVersion ASYNC_SEARCH_STATUS_SUPPORTS_KEEP_ALIVE = def(8_582_00_0);
+    public static final TransportVersion TIME_SERIES_ID_HASHING = def(8_582_10_0);
```

Comment:
You should not use that for a patch version -- those are reserved for serverless.
From the docs a few lines below:
```
     * M_NNN_SS_P
     *
     * M - The major version of Elasticsearch
     * NNN - The server version part
     * SS - The serverless version part. It should always be 00 here, it is used by serverless only.
     * P - The patch version part
```